monasca-log-api/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM python:2-alpine
```

Comment:
About that I had mixed feeling.
From one side I'd prefer reusing official images instead of manually installing Python.
On the other side my biggest concern is lack of possibility to easily build Python3 based container.

[monasca-api](https://github.com/openstack/monasca-api) does not have Py3 support, but log-api is different beast, it was much easier to have it implemented there. Not to mention that recently we've managed to enable monasca-notification (still some job is needed, but at least we won't have regression of failing tests). 

Having all that said, I think I will simply rollback to monasca-api approach here to enable that possibility.